snuba/datasets/transactions.py

```diff
@@ -182,6 +182,8 @@ def column_expr(self, column_name, query: Query, parsing_context: ParsingContext
             return 'IPv4NumToString(ip_address_v4)'
         if column_name == 'ip_address_v6':
             return 'IPv6NumToString(ip_address_v6)'
+        if column_name == 'ip_address':
+            return f"coalesce(IPv4NumToString(ip_address_v4), IPv6NumToString(ip_address_v6))"
```

Comment:
you do not need IPv6NumToString and IPv4NumToString. We upgraded the driver to access clickhouse.